bin/jobemissions

```diff
@@ -26,6 +27,28 @@ def round_to_nearest_half_hour(t: datetime) -> datetime:
    return t.replace(second=0, microsecond=0)
 
 CI_DATA_DIR = "/work/z19/shared/carbonintensity"
+SCOPE2_AND_3_EMISSIONS_DIR = "/Users/nattybatty/HPC_Emissions_project/emissions-tools/scripts/cleaned_nodes_filled_embodied.csv"
```

Comment:
Once we have the shared folder set up, we should put the data there and update this line - just adding this as a note for later so we don't forget